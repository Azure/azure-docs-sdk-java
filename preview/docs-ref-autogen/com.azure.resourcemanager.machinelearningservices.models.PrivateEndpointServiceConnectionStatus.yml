### YamlMime:JavaType
uid: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "Defines values for PrivateEndpointServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.REJECTED"
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.TIMEOUT"
methods:
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.machinelearningservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-machinelearningservices:1.0.0-beta.1
