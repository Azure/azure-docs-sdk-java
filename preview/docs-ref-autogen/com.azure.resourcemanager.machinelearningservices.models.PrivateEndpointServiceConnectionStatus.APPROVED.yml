### YamlMime:JavaMember
uid: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.machinelearningservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.machinelearningservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-machinelearningservices:1.0.0-beta.1
